tests/core/TransactionProcessor.spec.ts

```diff
@@ -19,9 +19,9 @@ import TransactionProcessor from '../../lib/core/versions/latest/TransactionProc
 import ValueTimeLockModel from '../../lib/common/models/ValueTimeLockModel';
 import ValueTimeLockVerifier from '../../lib/core/versions/latest/ValueTimeLockVerifier';
 
-describe('TransactionProcessor', () => {
-  const config = require('../json/config-test.json');
-  let casClient: Cas;
+describe('TransactionProcessor', async () => {
+  const config = await import('../json/config-test.json');
```

Comment:
await import?

Reply:
'coz import returns a promise.... I looked at all imports used this way in our codebase, they all await. Let me know if I am smocking something I shouldn't be!